helpers.sh

```diff
@@ -54,7 +54,13 @@ function repo_sync {
 		status "Adding git config extraheader for $domain"
 		git config --global http.https://${domain}.extraheader "$(cat /secrets/git.http.extraheader)"
 	fi
-	run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES}
+	for i in $(seq 4); do
+		run repo init --repo-rev=v2.29.4 --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES} && break
+		run sleep $(($RANDOM%20+10))
+		if [ $i -eq 4 ] ; then
+			exit 1
```

Comment:
Can you just add a print with a message saying why it is aborting the execution?

Reply:
The trace back is printed as I show on the commit message, I can print the return code for now and text time this happen I add the message descriptions of the return code.

Reply:
at this moment I don't know the reason and only the python trace back.

Reply:
fixed